syft_tensorflow/serde/serde_utils.py

```diff
@@ -0,0 +1,20 @@
+
+
+def history_numpy_to_float(history):
```

Comment:
if we end up using the numpy simplifier/detailer, this should be removed

Reply:
fixed, see comment above